src/Microsoft.AspNetCore.Diagnostics/DeveloperExceptionPage/Views/StackFrameInfo.cs

```diff
@@ -1,54 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
I'll work with folks to come up with official guidance on this. We definitely can't just remove this. But we can perhaps both document it as being obsolete, as well as mark it as `[Obsolete]` (*warning*, not *error*), to alert users. And then file a bug for 2.0.0 to delete it (we could actually go create that milestone if needed).